File: tests/go/cases/case_269_local_clone_named_retained_06.go
package escape_tests

var retainedCase269 = []map[string]string{}

func Case269LocalCloneNamedRetained06(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "local_clone_named_retained_06",
		"entity": "extreme",
		"stage": "stress",
		"input": raw,
		"checksum": "local_clone_named_retained_06:" + raw,
	}
	retainedLocal := []map[string]string{payload}
	_ = retainedLocal
	// SAFE: misleading local name only.
	return payload["checksum"]
}

File: tests/nodejs/cases/case_269_local_clone_named_retained_06.js
/**
 * Task case 269: local_clone_named_retained_06 deep stress pattern.
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case269LocalCloneNamedRetained06(input) {
  const taskName = 'local_clone_named_retained_06';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'extreme',
    stage: 'stress',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  const retainedLocal = [payload];
  void retainedLocal.length;
  // SAFE: local retained* name only.
  return payload.checksum;
}

module.exports = { case269LocalCloneNamedRetained06 };
